CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,96 @@
+- Remove the prefix `rancher-wins` when collecting antrea-agent logs on Windows. ([#6223](https://github.com/antrea-io/antrea/pull/6223), [@wenyingd])
+- Upgrade K8s libraries to v0.29.2. ([#5843](https://github.com/antrea-io/antrea/pull/5843), [@hjiajing])
+- Upgrade base image from UBI8 to UBI9 for Antrea UBI images. ([#5737](https://github.com/antrea-io/antrea/pull/5737), [@xliuxu])
+
+### Fixed
+
+- Fix nil pointer dereference when `ClusterGroup`/`Group` is used in NetworkPolicy controller. ([#6077](https://github.com/antrea-io/antrea/pull/6077), [@tnqn])
+- Disable `libcapng` to make logrotate run as root in UBI images to fix an OVS crash issue. ([#6052](https://github.com/antrea-io/antrea/pull/6052), [@xliuxu])
+- Fix a race condition in antrea-agent Traceflow controller when a tag is associated again with a new Traceflow before the old Traceflow deletion event is processed. ([#5954](https://github.com/antrea-io/antrea/pull/5954), [@tnqn])
+- Change the maximum flags from 7 to 255 to fix the wrong TCP flags validation issue in `Traceflow` CRD. ([#6050](https://github.com/antrea-io/antrea/pull/6050), [@gran-vmv])
+- Use 65000 MTU upper bound for interfaces in `encap` mode to account for the MTU automatically configured by OVS on tunnel ports, and avoid packet drops on some clusters. ([#5997](https://github.com/antrea-io/antrea/pull/5997), [@antoninbas])
+- Install multicast related iptables rules only on IPv4 chains to fix the antrea-agent initialization failure occurred when the Multicast feature is enabled in dual-stack clusters. ([#6123](https://github.com/antrea-io/antrea/pull/6123), [@wenyingd])
+- Remove incorrect AntreaProxy warning on Windows when `proxyAll` is disabled. ([#6242](https://github.com/antrea-io/antrea/pull/6242), [@antoninbas])
+- Explicitly set kubelet's log files in Prepare-Node.ps1 on Windows, to ensure that they are included in support bundle collections. ([#6221](https://github.com/antrea-io/antrea/pull/6221), [@wenyingd])
+- Add validation on antrea-agent options to fail immediately when encryption is requested and the Multicast feature enabled. ([#5920](https://github.com/antrea-io/antrea/pull/5920), [@wenyingd])
+- Don't print the incorrect warning message when users run `antrea-controller --version` outside of K8s. ([#5993](https://github.com/antrea-io/antrea/pull/5993), [@prakrit55])
+- Record event when EgressIP is uninstalled from a Node and remains unassigned. ([#6011](https://github.com/antrea-io/antrea/pull/6011), [@jainpulkit22])
+- Fix a bug that the local traffic cannot be identified on `networkPolicyOnly` mode. ([#6251](https://github.com/antrea-io/antrea/pull/6251), [@hongliangl])
+- Use reserved OVS controller ports for the default Antrea ports to fix a potential `ofport` mismatch issue. ([#6202](https://github.com/antrea-io/antrea/pull/6202), [@antoninbas])
+
+
+
```

Comment:
```suggestion
```

Reply:
Should we left only one line here?

Reply:
I will push the change later after all tests passed.